lib/libvmod_blob/vmod_blob.c

```diff
@@ -31,11 +31,21 @@
 
 #include "cache/cache.h"
 
-#include "vcc_if.h"
 #include "vmod_blob.h"
 
 #define VMOD_BLOB_TYPE 0xfade4faa
 
+static const struct vmod_blob_codec *codecs[] = {
```

Comment:
Moving the codecs into the various sources, just calling their methods and properties from this table, is a good improvement for encapsulation.